ansible/roles/passenger/files/etc/nginx/nginx.conf

```diff
@@ -19,8 +19,7 @@ http {
   keepalive_timeout 65;
   types_hash_max_size 2048;
   server_tokens off;
-  more_clear_headers Server;
-  more_clear_headers X-Powered-By;
+  passenger_show_version_in_header off;
```

Comment:
Are you ready for the nitpick of the day 😅 Should we move this to the passenger section of the config below?

Reply:
My day is going really average and I shall.